app/src/main/java/com/example/arvin/nepaltouristguide/camping/CampingActivity.java

```diff
@@ -72,4 +76,9 @@ protected void onDestroy() {
         super.onDestroy();
     }
 
+    @Override
```

Comment:
move this method above ondestory
methods should be in order like reading a page

Reply:
similar methods